app/graph/types/team_type.rb

```diff
@@ -224,7 +224,7 @@ def sources(keyword: nil)
   end
 
   def tag_texts(keyword: nil)
-    object.tag_texts_by_keyword(keyword)
+    object.tag_texts_by_keyword(keyword).sort_by { |tag_text| tag_text.text.downcase }
```

Comment:
In this case, we want an `ActiveRecord::Collection` to be returned, this way, the GraphQL pagination can be applied. If we use `sort_by`, it's going to convert the query into an array, which is going to load all tags in memory, creating a performance issue and breaking pagination. For this case, the right way is: `.order('text ASC')`. See that the underlying method returns an `ActiveRecord::Collection`: https://github.com/meedan/check-api/blob/develop/app/models/team.rb#L430. So, we can just append other `ActiveRecord` methods to the chain (order, limit, where, etc.).

Reply:
Oh, that makes sense, thanks! Silly me and copilot! :D